packages/suite/src/constants/suite/experimental.ts

```diff
@@ -1,15 +1,23 @@
 import { TranslationKey } from '@suite-common/intl-types';
+import { isDesktop } from '@trezor/env-utils';
 
-export enum ExperimentalFeature {
-    PasswordManager = 'password-manager',
-}
+export type ExperimentalFeature = 'password-manager' | 'silent-update';
```

Comment:
I thought using enums is encouraged now, so I'm actually glad for this change of yours, it's definitely more practical. :+1: 

But, given that `ExperimentalFeatureConfig` is now required for every feature, can we make `EXPERIMENTAL_FEATURES` single source of truth, maybe like this?
```
export const EXPERIMENTAL_FEATURES = {
   ...
} satisfies Record<string, ExperimentalFeatureConfig>;

// I've never used satisfies before (♥‿♥)

export type ExperimentalFeature = keyof typeof EXPERIMENTAL_FEATURES;
```

Reply:
> I thought using enums is encouraged now,

I don't know. My current understanding is that Enums are pain in Javascript. Do you have any source for this? :pray:

Reply:
By doing that inference you lose the type-safety of `EXPERIMENTAL_FEATURES[feature]`. In your proposal this would satisfy any `string`. In current implementation only `'password-manager' | 'silent-update'` will.

Reply:
Nope?
![image](https://github.com/user-attachments/assets/c4da4792-c919-4500-a9e2-7759b4734c75)

Reply:
Ah ok, you got me. Didnt know that satisfies works this way. Cool! Thanks!

Reply:
fixed